src/component/description/cases/observing/SettingModal.js

```diff
@@ -16,20 +28,14 @@ const SettingModal = ({
   handleCheck,
   showGrid,
   handleChangeComplete,
+  handleLang,
```

Comment:
Should be `handleChangeLanguage` or `handleChangeLang`, because `handleLang` does not describe the action correctly.